app-emulation/vagrant/vagrant-2.2.4.ebuild

```diff
@@ -29,6 +29,7 @@ RDEPEND="${RDEPEND}
 ruby_add_rdepend "
 	>=dev-ruby/bcrypt_pbkdf-1.0.0
 	>=dev-ruby/childprocess-0.6.0
+	ruby_targets_ruby26? ( >=dev-ruby/childprocess-1.0.1-r1 )
```

Comment:
don't do that. eclass and portage already take care of that, as it's not pure `RDEPEND`.
real `RDEPEND` is generated and looks like this
```
...
>=dev-ruby/childprocess-0.6.0[ruby_targets_ruby26]
>=dev-ruby/net-sftp-2.1[ruby_targets_ruby26]
...
```